TodoApp/test/TodoApp.Api.Tests/Services/UrlGeneratorTests.test.cs

```diff
@@ -35,5 +29,15 @@ public void GetItemUrl_UrlReceived()
 
             Assert.That(receivedUrl, Is.EqualTo(requestedUrl));
         }
+
+        private static bool ContainsCorrectId(object first)
```

Comment:
Great you did not name the function `GetAnonymousArg` as commit message states 😉

Reply:
I don't like `first` parameter name - choose a more descriptive one - how is the second (😉) argument called in the `urlHelper.Route` method? That is the argument you resolve upon, isn't it?